packages/angular_devkit/build_angular/plugins/webpack/analytics.ts

```diff
@@ -158,7 +158,7 @@ export class NgBuildAnalyticsPlugin {
 
       // Count the number of `Component({` strings (case sensitive), which happens in __decorate().
       // This does not include View Engine AOT compilation, we use the ngfactory for it.
-      this._stats.numberOfComponents += countOccurrences(module._source.source(), ' Component({');
+      this._stats.numberOfComponents += countOccurrences(module._source.source(), 'Component({');
```

Comment:
Just to clarify, the beginning space was the issue?

Reply:
Yes